### PR TITLE
{vis}[GCCcore/8.3.0,foss/2019b] PyOpenGL v3.1.5, PyQtGraph v0.11.0 w/ Python 2.7.16 + 3.7.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1dc17033a8404e19d451cb762bdb8792 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 3 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/28944e8916017116e1cfe070c4a9e48b for a full test report.

**Comment by lexming** [comment]
The cause of the failed test is not related to this PR

**Comment by verdurin** [comment]
@lexming Did you spot what was wrong so I can fix it and run the test properly?

**Comment by lexming** [comment]
@verdurin not really, I only see in the log that `PyQt5` failed during the installation of the `PyQtWebEngine` component because an import of `PyQt5.QtCore` failed. `PyQt5.QtCore` should have been installed by the previous component, but the partial log does not show what happened at that stage.

**Comment by migueldiascosta** [comment]
I get the same error when building `PyQt5`. `PyQt5.QtCore` is built, but if I then try to import it manually I get `undefined symbol: _ZN8qfloat1613mantissatableE`

**Comment by migueldiascosta** [comment]
ah, https://github.com/easybuilders/easybuild-easyconfigs/pull/10755 says there's a patch missing in the `Qt5` installation, will reinstall it

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/6ae7b1c1b43c0a0da03aa415430d2961 for a full test report.

**Comment by migueldiascosta** [comment]
@verdurin do you want to try resubmitting the test report after rebuilding `Qt5`?

**Comment by lexming** [comment]
@migueldiascosta that's funny :laughing: I just went through the same issue a month ago in that PR. I knew that this error sounded familiar, but I could not remember why.

**Comment by migueldiascosta** [comment]
@lexming I hardly remember what I had for dinner yesterday...

**Comment by verdurin** [comment]
@migueldiascosta yes, thanks for ferreting it out - rebuild underway now.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/57201df34f2726ef67477fabfaffcbef for a full test report.

File: easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ('binutils', '2.32'),
+    ('bzip2', '1.0.8'),
+]
+
+dependencies = [
+    ('Mesa', '19.1.7'),
+]
+
+download_dep_fail = True
+use_pip = True
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb:38]: Aren't some of these redundant now with recent changes to the `easyblock`, or is that just for `PythonBundle`?

File: easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.11.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),  # required for numpy
+    ('PyQt5', '5.13.2', versionsuffix),
+    ('PyOpenGL', '3.1.5'),
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.11.0-foss-2019b-Python-3.7.4.eb:26]: Same as above.

File: easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb:38]: what changes do you refer to?

Comment (lexming) [reply easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb:38]: @verdurin ping

Comment (verdurin) [reply easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb:38]: @lexming I thought that at least some of those parameters were now the default. Could be wrong.

Comment (lexming) [reply easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb:38]: @verdurin AFAIK there have been no changes to `PythonPackages` in that regard.

File: easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.11.0-foss-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (jhein32) [reply easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.11.0-foss-2019b-Python-3.7.4.eb:26]: May I ask why this is stalled?  Are here real issues?  I need the PyOpenGL for another build project.

**Comment by jhein32** [comment]
May I ask why this is stalled? Are here real issues? I need the PyOpenGL for another build project.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**